ci/build/build-manylinux-wheel.sh

```diff
@@ -10,6 +10,7 @@ cd python
 # Set the commit SHA in _version.py.
 if [[ -n "$TRAVIS_COMMIT" ]]; then
   sed -i.bak "s/{{RAY_COMMIT_SHA}}/$TRAVIS_COMMIT/g" ray/_version.py && rm ray/_version.py.bak
+  sed -i.bak "s/{{RAY_COMMIT_SHA}}/$TRAVIS_COMMIT/g" ../src/ray/common/constants.h && rm ../src/ray/common/constants.h.bak
```

Comment:
you don't really want this.. this will invalidate the bazel cache and make wheel building much slower for your team.

Reply:
you would want to instead read a stamp file in the wheel on loading or something, that is not blended in the c++ build process like this.

Reply:
or let the launcher of gcs / raylet send in a flag value.